core/src/banking_stage/scheduler_messages.rs

```diff
@@ -0,0 +1,54 @@
+use {
+    crate::immutable_deserialized_packet::ImmutableDeserializedPacket,
+    solana_poh::poh_recorder::Slot, solana_sdk::transaction::SanitizedTransaction, std::sync::Arc,
+};
+
+/// A unique identifier for a transaction batch.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub struct TransactionBatchId(u64);
+
+impl TransactionBatchId {
+    pub fn new(index: u64) -> Self {
+        Self(index)
+    }
+}
+
+/// A unique identifier for a transaction.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
+pub struct TransactionId(u64);
+
+impl TransactionId {
+    pub fn new(index: u64) -> Self {
+        Self(index)
+    }
+}
+
+/// Message: [Scheduler -> Worker]
+/// Transactions to be consumed (executed, recorded, committed)
+pub struct ConsumeWork {
+    pub batch_id: TransactionBatchId,
+    pub transaction_ids: Vec<TransactionId>,
+    pub transactions: Vec<SanitizedTransaction>,
+    pub max_age_slots: Vec<Slot>,
```

Comment:
following a discussion with @ryoqun, we shouldn't let `SanitizedTransaction`s pass epoch boundaries - so we can mark txs (at sanitize time) with a max age slot it will expire at.

Workers should check this before execution, and quickly disregard txs if the bank the worker is using `> max_age_slot`